scripts/indexing/developer_docs_connector.py

```diff
@@ -0,0 +1,210 @@
+from typing import Union, Sequence, List
+from collections import Counter
+
+from glean.indexing.connectors import BaseDatasourceConnector
+from glean.indexing.models import (
+    ContentDefinition,
+    CustomDatasourceConfig,
+    DocumentDefinition,
+    UserReferenceDefinition
+)
+from glean.api_client.models import (
+    ObjectDefinition,
+    PropertyDefinition,
+    UIOptions,
+    PropertyType,
+    DatasourceCategory,
+    CustomProperty,
+    DocumentPermissionsDefinition
+)
+from data_types import PageInfoData, ApiReferenceData
+
+class CustomDeveloperDocsConnector(BaseDatasourceConnector[Union[PageInfoData, ApiReferenceData]]):
+    configuration: CustomDatasourceConfig = CustomDatasourceConfig(
+        name="customDeveloperDocsDatasource",
+        display_name="Custom Glean Developer Docs",
+        datasource_category=DatasourceCategory.KNOWLEDGE_HUB,
+        url_regex="https://developers.glean.com/.*",
+        icon_url="https://glean-public-external-be.glean.com/api/v1/images?key=eyJ0eXBlIjoiVUdDIiwiaWQiOiIwIiwiZHMiOiJHQUxMRVJZLUlNQUdFLVBJQ0tFUiIsImNpZCI6ImVmNzI4NWM1LWY0ZjgtNDZhYi05NGQ3LTAxMjc4ZjhmNTAwMyIsImV4dCI6Ii5wbmcifQ==",
```

Comment:
This works, but the image may change and that may break in our indexing. If you base64 encode this, you can pass that in instead.


```suggestion
        icon_url="ata:image/png;base64,iVBORw0KGgoAAAANSUhEU....."
```